data/saas/config/mailchimp_config.yml

```diff
@@ -9,6 +9,10 @@ saas_config:
     - name: domain
     - name: username
     - name: api_key
+    - name: transactional_domain
+      default_value: mandrillapp.com/api/1.0
+    - name: transactional_api_key
+      default_value:
```

Comment:
Seems like all connector params are required unless there's a default value, but only a set will apply for consent requests, and a different set for access and erasure.  The user may not have both sets of credentials.

Thinking about what is the best path forward here.  We could work on making some optional, we could have a separate "mailchimp consent saas config", other options?

Reply:
I'm thinking a separate "Mailchimp Transactional" saas connector for now..